isofit/test/test_cli.py

```diff
@@ -50,11 +48,10 @@ def surface(cube_example):
 
     # Generate the surface.mat using the image_cube example config
     # fmt: off
-    isofit = pathlib.Path(root)
     surface_model(
-        config_path     = str(isofit / "examples/20171108_Pasadena/configs/ang20171108t184227_surface.json"),
```

Comment:
`isofit.root` points to the directory containing the `isofit` installation, which isn't a reliable path, and I would like to suggest the attribute be removed entirely. At this point the remaining uses are:

```console
$ git grep "isofit.*root" "*.py"
isofit/core/geometry.py:            isofit.root, "data", "earth_sun_distance.txt"
isofit/radiative_transfer/radiative_transfer_engine.py:        isofit.root, "data", "earth_sun_distance.txt"
isofit/utils/downloads.py:output = partial(click.option, "-o", "--output", default=isofit.root, show_default=True)
```

We already know that using `isofit.root` to find `earth_sun_distance.txt` is not valid for a `$ pip install`-ed version of `isofit`. The remaining use is by `$ isofit download`, which fetches `earth_sun_distance.txt`, and can easily be altered. It would be reasonable for `$ isofit download` to write to the current working directory by default, so ISOFIT developers could fetch these files as part of the development environment setup:

```console
$ git clone ...
$ cd isofit/
$ python3 -m venv venv
$ ./scripts/setup-devenv.sh venv
$ source venv/bin/activate
$ (venv) isofit download
```

The `$ isofit download` step could also be included in `setup-devenv.sh`, but it seems like some config files would also need to be updated.

I recall a discussion with the ISOFIT team about how many (most? all?) users of ISOFIT inside of JPL install ISOFIT via `$PYTHONPATH`, but I think this is encouraging development patterns that make installing the package via `$ pip` and `$ conda` difficult. I think my recent CI work makes these issues more apparent, and it seems like `isofit.root` and the files like `earth_sun_distance.txt` are probably the two remaining issues related to packaging. It seems `earth_sun_distance.txt` should not rely on `isofit.root`, which to me means that attribute has no reason to exist. And again, for users who `$ pip install`-ed ISOFIT, `isofit.root` points to Python's `site-packages` directory.

Reply:
@pl-kevinwurster You're absolutely right. This all points to the issues we had in the past of utilizing data and example files from a pip or conda install of ISOFIT. I'm also highly in favor of not having data files in Python's `site-packages` directory, so let's handle a potential removal of `isofit.root` in a separate PR.